packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -3435,19 +3177,20 @@ function startWorkOnPendingInteractions(
   }
 }
 
-function finishPendingInteractions(root, committedExpirationTime) {
+function finishPendingInteractions(root, committedLanes) {
   if (!enableSchedulerTracing) {
     return;
   }
 
-  const earliestRemainingTimeAfterCommit = root.firstPendingTime_opaque;
+  const remainingLanesAfterCommit = root.pendingLanes;
 
   let subscriber;
 
   try {
     subscriber = __subscriberRef.current;
     if (subscriber !== null && root.memoizedInteractions.size > 0) {
-      const threadID = computeThreadID(root, committedExpirationTime);
+      // FIXME: More than one lane can finish in a single commit.
```

Comment:
How often do you think this will break things?

Reply:
I wasn’t sure about this. If it’s not already a problem then this doesn’t make it worse, those, since multiple expirations times could finish in the same batch, too.